string move_req
int16  pose_req 
---
int16 move_resp